### ISSUE TITLE
Support multiple vm and storage checks

### ISSUE THREAD

**Issue by nbuchwitz** [nbuchwitz/check_pve#5]
The check command should support more than one resource name parameter, to allow checks for more than one storage or vm per check execution.

**Comment by fulljackz** [comment]
Hi,

Is there any news about this feature ? I do not think so.
It would be a very good thing to get metrics for all running vms, dynamically.

I am going to try something in the script.

**Comment by nbuchwitz** [comment]
Unfortunately no news about this, but it is still on the agenda.

**Comment by hamapj** [comment]
Hi, is there any progress about this?

I would highly appreciate the idea mentioned by @fulljackz to be able to dynamically retriev metrics, maybe via specifing the resource type?

I already took a look at the code and would gladly create a PR for this.

**Comment by nbuchwitz** [comment]
No, haven't had any capacity for this. Happy to review a pr and collaborate on the integration.

**Comment by hamapj** [comment]
I have created a PR [here](https://github.com/nbuchwitz/check_pve/pull/46) for this issue.
